README.md

```diff
@@ -8,12 +8,61 @@ This plugin contains rule exclusions for [Nextcloud](https://nextcloud.com/), a
 
 For full and up to date instructions for the different available plugin installation methods, refer to [How to Install a Plugin](https://coreruleset.org/docs/concepts/plugins/#how-to-install-a-plugin) in the official CRS documentation.
 
+## Increasing max upload size
+
+Large uploads can be modified with SecRequestBodyLimit. Or they can be more controlled by using the following:
+
+```
+SecRule REQUEST_URI "@endsWith /index.php/apps/files/ajax/upload.php" \
+    "id:9508610,\
+    phase:1,\
+    t:none,\
+    nolog,\
+    ctl:requestBodyLimit=1073741824"
+```
+
+ctl:requestBodyLimit is not supported in libmodsecurity3, Nginx users can increase max upload size
```

Comment:
What about Apache users?

Reply:
They're covered by `requestBodyLimit`.

Reply:
But it says that this isn't supported by libmodsecurity3.

Reply:
Oh, sorry. My bad. You were referring to httpd on libmodsecurity 3. I always forget that's a thing 😅

Reply:
it doesn't look like libmodsecurity3 apache is still supported, no commits in 2.5 years and it's in beta?

Reply:
I've added an example nonetheless